socketio/src/client/client.rs

```diff
@@ -266,22 +338,19 @@ mod test {
 
         // waiting for client to emit messages
         std::thread::sleep(Duration::from_millis(100));
-        let mut pre_num = 0;
-        {
-            let packets = packets.read().unwrap();
-            pre_num = packets.len();
-        }
+        let _packets = packets.read().unwrap();
+        let pre_num = _packets.len();
+        drop(_packets);
 
-        socket.disconnect();
+        let _ = socket.disconnect();
         socket.reconnect();
 
         // waiting for client to emit messages
         std::thread::sleep(Duration::from_millis(100));
-        let mut post_num = 0;
-        {
-            let packets = packets.read().unwrap();
-            post_num = packets.len();
-        }
+
+        let _packets = packets.read().unwrap();
+        let post_num = _packets.len();
+        drop(_packets);
```

Comment:
nit: _ prefix misleading
```suggestion
        let packets = packets.read().unwrap();
        let post_num = packets.len();
        drop(packets);
```